docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -319,17 +377,58 @@ variables; worker nodes **must** specify the `CUBESTORE_WORKER_PORT` and
 `CUBESTORE_META_ADDR` environment variables. More information about these
 variables can be found [in the Environment Variables reference][ref-config-env].
 
+A sample Docker Compose stack setting this up might look like:
+
+```yaml
+version: '2.2'
+services:
+  cubestore_router:
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=localhost
+      - CUBESTORE_META_PORT=9999
+      - CUBESTORE_WORKERS=localhost:9001,localhost:9002
+  cubestore_worker_1:
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=localhost
+      - CUBESTORE_WORKER_PORT=9001
+      - CUBESTORE_META_ADDR=localhost:9999
+  cubestore_worker_2:
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=localhost
+      - CUBESTORE_WORKER_PORT=9002
+      - CUBESTORE_META_ADDR=localhost:9999
+  cube:
+    image: cubejs/cube:latest
+    ports:
+      - 4000:4000
+    environment:
+      - CUBEJS_CUBESTORE_HOST=localhost
+      - CUBEJS_CUBESTORE_PORT=9999
+    depends_on:
+      - cubestore_router
+    links:
```

Comment:
Let's remote the `links` section altogether.
Containers are linked together by default, this only seems to complicate things.